projects/packages/jetpack-mu-wpcom/changelog/bump-jetpack-mu-wpcom-4-8-0-alpa

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Bump jetpack-mu-wpcom to 4.8.0-alpha
```

Comment:
Chances are you don't actually want the changelog to say "Bump jetpack-mu-wpcom to some version". So do it like this:
```suggestion
Comment: Bump jetpack-mu-wpcom to 4.7.1-alpha

```